bridge/mumble/mumble.go

```diff
@@ -0,0 +1,259 @@
+package bmumble
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strconv"
+	"time"
+
+	"layeh.com/gumble/gumble"
+	"layeh.com/gumble/gumbleutil"
+
+	"github.com/42wim/matterbridge/bridge"
+	"github.com/42wim/matterbridge/bridge/config"
+	"github.com/42wim/matterbridge/bridge/helper"
+	stripmd "github.com/writeas/go-strip-markdown"
+
+	// We need to import the 'data' package as an implicit dependency.
+	// See: https://godoc.org/github.com/paulrosania/go-charset/charset
+	_ "github.com/paulrosania/go-charset/data"
+)
+
+type Bmumble struct {
+	client             *gumble.Client
+	Nick               string
+	Host               string
+	Channel            *uint32
```

Comment:
0 is the ID of the root channel, so this can't be used as "unset" here. Regarding possible panics: This value is accessed in few cases only, all of which have a nil check.

Reply:
ok makes sense